mobile-app/app/screens/AppNavigator/screens/Auctions/components/BatchCard.tsx

```diff
@@ -77,200 +84,188 @@ export function BatchCard(props: BatchCardProps): JSX.Element {
+                    "flex flex-row items-center rounded-2xl-v2 py-2 px-3"
+                  )}
+                  onPress={onQuickBid}
+                  testID={`${testID}_quick_bid_button`}
                 >
-                  {value}
-                </ThemedText>
-              )}
-              thousandSeparator
-              value={totalCollateralsValueInUSD}
-            />
+                  <SymbolIcon
+                    symbol={batch.loan.displaySymbol}
+                    styleProps={tailwind("w-4 h-4")}
+                  />
+                  <ThemedTextV2
+                    style={tailwind(
+                      "font-semibold-v2 text-xs text-center ml-1"
+                    )}
+                  >
+                    {translate(
+                      "components/BatchCard",
+                      batch?.froms.includes(address)
+                        ? "Quick rebid"
+                        : "Quick bid"
+                    )}
+                  </ThemedTextV2>
+                </ThemedTouchableOpacityV2>
+              </View>
+            </View>
           </View>
+          <VerticalProgressBar
+            height={progressBarHeight}
+            normalizedBlocks={normalizedBlocks.toNumber()}
+            color={timeRemainingThemedColor}
+          />
         </View>
-        <MinNextBidTextRow
-          displaySymbol={batch.loan.displaySymbol}
-          minNextBidInToken={minNextBidInToken}
-          minNextBidInUSD={minNextBidInUSD}
-          testID={`batch_${batch.index}_min_next_bid`}
-        />
       </TouchableOpacity>
-
-      <AuctionTimeProgress
-        liquidationHeight={vault.liquidationHeight}
-        blockCount={blockCount}
-        label="Auction time remaining"
-      />
-      <BatchCardButtons
-        onPlaceBid={onPlaceBid}
-        onQuickBid={onQuickBid}
-        testID={testID}
-      />
-    </ThemedView>
+    </ThemedViewV2>
   );
 }
 
-const BatchCardInfo = memo(
-  (props: {
-    iconName: IconName;
-    text: string;
+const BidInfo = memo(
+  ({
+    isHighestBidder,
+    hasFirstBid,
+    testID,
+    isOutBid,
+  }: {
+    isHighestBidder: boolean;
+    hasFirstBid: boolean;
+    isOutBid: boolean;
     testID: string;
   }): JSX.Element => {
+    const { isLight } = useThemeContext();
+    const { testId, title, icon, color } = useMemo(() => {
+      if (isHighestBidder) {
+        return {
+          testId: `${testID}_lost`,
+          title: "Leading bid",
+          icon: "check-circle",
+          color: "text-green-v2",
+        };
+      }
+      if (isOutBid) {
+        return {
+          testId: `${testID}_leading`,
+          title: "Outbid",
+          icon: "info",
+          color: "text-orange-v2",
+        };
+      }
+      return {
+        testId: hasFirstBid ? `${testID}_next_bid` : `${testID}_no_bid`,
+        title: "Min bid",
+        color: isLight ? "text-mono-light-v2-900" : "text-mono-dark-v2-900",
+      };
+    }, []);
```

Comment:
toggle between theme mode now will not update the text color
```suggestion
    }, [isLight]);
```